tests/src/test/java/features/frontendfeatures/landingPage.feature

```diff
@@ -1,11 +1,17 @@
-Feature: Access landing page and verify title
-
-  Background:
-    Given browser is initialized
+Feature: User can navigate to the efiling landing page
 
   @frontend
-  Scenario: Verify if the landing page is available
-    Given user is navigated to the landing page
-    Then page title is verified
-    And browser is closed
-
+  Scenario: Verify valid account guid redirects efiling form page
+    Given user is on the landing page
+    When  user enters invalid account guid "5EDDA22161C043AB9D508158BF4A7828"
```

Comment:
are we comfortable storing this in the repo?

Reply:
Added a placeholder message to input the test data now. Working on a solution to pass it dynamically from the external data source. Will be part of the next update. Thanks.